### ISSUE TITLE
Fine-tuning only the bias-terms

### ISSUE THREAD

**Issue by matteo-grella** [deepset-ai/FARM#695]
To speed up fine tuning without sacrificing accuracy, I propose to add a built-in option to freeze most of the
parameters and fine-tuning only the bias-terms.

Reference:
[BitFit: Simple Parameter-efficient Fine-tuning for Transformer-based Masked Language-models](https://nlp.biu.ac.il/~yogo/bitfit.pdf)

**Comment by Timoeller** [comment]
Wow, this is an interesting approach indeed. Thanks for sharing @matteo-grella 

Would you like to implement the functionality? I guess starting at something simple like classification?
If you continue, please have a look at #141 for some help to get you kickstarted.

**Comment by Timoeller** [comment]
I still think it is a great idea, I will move this issue into our icebox because of inactivity.

**Comment by matteo-grella** [comment]
@Timoeller I agree with you. However, I attempted to implement that in FARM for text classification and I got poor results :-|

**Comment by Timoeller** [comment]
Hah interesting, there is no free lunch after all again. Which dataset did you use?

And thanks for reporting back, lets leave the issue in the icebox and warm it up if other people get excited about bias only finetuning for their datasets : )

**Comment by Jordy-VL** [comment]
I know this is stale, but did it end up being implemented somewhere? Would love to try this and prefer to not start from zero :p Thanks a lot!
